src/index.html

```diff
@@ -9,6 +9,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header__big">
+      <div class="logo__div">
+        <a href="#logo" class="moyologo nav__link ">
+          <img src="images/logo.png" alt="moyologo">
+        </a>
+      </div>
+      <nav class="navigation__big">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="№APPLE" class=" is-active nav__link">apple</a>
```

Comment:
Якірне посилання задається через решітку
```suggestion
            <a href="#APPLE" class=" is-active nav__link">apple</a>
```